zipline/transforms/batch_transform.py

```diff
@@ -33,13 +33,7 @@
 )
 
 from zipline.utils.data import MutableIndexRollingPanel
-from zipline.protocol import Event
-from zipline.finance.trading import TradingEnvironment
-
-# HACK the BatchTransform module stores a trading environment to be used by
-# the transforms
-# TODO remove this hack, if not this whole module
-_batch_transform_env = TradingEnvironment()
+from zipline.protocol import Event, get_algo_instance
```

Comment:
Surprised to see get_algo_instance imported from protocol instead of zipline.utils.algo_instance, but not a big deal.